src/region_labelling.rs

```diff
@@ -128,8 +128,8 @@ where
     I::Pixel: Eq,
 {
     let (width, height) = image.dimensions();
-    let image_size = width as usize * height as usize;
-    if image_size >= 2usize.pow(32) {
+    let image_size: usize = width as usize * height as usize;
```

Comment:
Why have you added the type annotation here?

Reply:
It was mostly be making sure the types were correct; I've removed it now as it's unnecessary.